reference/apm/cloud/apm-settings.md

```diff
@@ -47,330 +47,12 @@ If a setting is not supported by {{ech}}, you will get an error message when you
 
 ## Supported standalone APM settings (legacy) [ec-apm-settings]
 
-{{ech}} supports the following setting when running APM in standalone mode (legacy).
+{{ech}} supports the settings listed in [APM documentation](/solutions/observability/apm/configure-apm-server.md) under "APM Server binary" when running APM in standalone mode (legacy). For versions before 9, refer to [older documentation](https://www.elastic.co/guide/en/observability/8.x/apm-configuring-howto-apm-server.html).
```

Comment:
@florent-leborgne is there a way to easily link to old (non-docsv3) docs from docs v3?

Reply:
Well, in this case they are all "There are no other versions available for this page." which isn't super helpful. I've updated the link to 8.18 docs. While I don't love it, it seems to be the most user friendly way